CrossCheckStata.do

```diff
@@ -2,16 +2,67 @@ clear
 clear matrix
 set mem 1g
 set more off
-insheet using CrossCheckBalanced1600.csv
+insheet using CrossCheckBalanced1000.csv
 xtset hhid tid
 capture log close
 log using CrossCheckStata.txt, replace
-*timer on 1
+*XTREG
+timer clear 
+timer on 1
 xi: xtreg y x1 x2 i.tid [aw=w], fe
-*timer off 1
+timer off 1
+timer list 
+timer clear 
+outreg2 using XTRegTestSTATA1000.doc, replace ctitle(Xtreg) keep(x1 x2) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
 * Fixed effects, robust
-timer on 2
+timer on 1
 xi: xtreg y x1 x2 i.tid [aw=w], fe vce(robust)
-timer off 2
-timer list
-log close
+timer off 1
+timer list 
+timer clear 
+outreg2 using XTRegTestSTATA1000.doc, append ctitle(Xtreg-R) keep(x1 x2) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1)) 
+timer clear 
+*Change specification
+timer on 1
+xi: xtreg y x1 i.tid [aw=w], fe
+timer off 1
+timer list 
+timer clear 
+outreg2 using XTRegTestSTATA1000.doc, append ctitle(Xtreg-one C) keep(x1) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
+timer on 1
+xi: xtreg y x1 i.tid [aw=w], fe
+timer off 1
+timer list 
+timer clear 
+outreg2 using XTRegTestSTATA1000.doc, append ctitle(Xtreg-one C-R) keep(x1) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
+
+*REGHDFE
+timer on 1
+reghdfe y x1 x2, absorb(hhid tid)
+timer off 1
+timer list 
+timer clear 
+outreg2 using ReghdfeTestSTATA1000.doc, replace ctitle(Reghdfe) keep(x1 x2) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
+* Fixed effects, robust
+timer on 1
+reghdfe y x1 x2, absorb(hhid tid) vce(cluster hhid)
+timer off 1
+timer list 
+timer clear 
+outreg2 using ReghdfeTestSTATA1000.doc, append ctitle(Reghdfe-R) keep(x1 x2) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
+*change specification
+timer on 1
+reghdfe y x1, absorb(hhid tid)
+timer off 1
+timer list 
+timer clear 
+outreg2 using ReghdfeTestSTATA1000.doc, append ctitle(Reghdfe-one C) keep(x1) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
+* Fixed effects, robust
+timer on 1
+reghdfe y x1, absorb(hhid tid) vce(cluster hhid)
+timer off 1
+timer list 
+timer clear 
+outreg2 using ReghdfeTestSTATA1000.doc, append ctitle(Reghdfe-one C-R) keep(x1) addtext(Group FE, YES, Time FE, YES) addstat(Running Time, r(t1))
+
+log close
```

Comment:
Watch out! Stata needs a newline at the end of every do file. Otherwise this line won't run.